storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1706,13 +1729,55 @@ static MYSQL_SYSVAR_UINT(
     RDB_DEFAULT_TBL_STATS_SAMPLE_PCT, /* everything */ 0,
     /* max */ RDB_TBL_STATS_SAMPLE_PCT_MAX, 0);
 
+static MYSQL_SYSVAR_UINT(table_stats_recalc_threshold_pct,
+                         rocksdb_table_stats_recalc_threshold_pct,
```

Comment:
Formatting here looks to be inconsistent, is it?

Reply:
Maybe it is but my policy is to keep a minimal diff with upstream to minimize conflicts. Therefore I prefer to wait until upstream will fix/change it. 
It seems that they do `clang-formatting` for entire MyRocks every few months. In the meantime MyRocks could be not `clang-format` valid.

Reply:
OK, this is not how we handled MyRocks in the past, we want _our_ tree to be as perfect as possible even if it causes some merge difficulties.  If we are sure that upstream reformats regularly, then we might consider relaxing.  Ideally upstream should catch format errors before accepting patches.